setup.py

```diff
@@ -48,15 +48,14 @@
     include_package_data=True,
     zip_safe=False,
     license='MPL-2.0',
+    python_requires=">=3.5",
```

Comment:
Yes, perhaps. My preference is to introduce changes in the preferred location rather than to cling to old conventions, but I also see the value in keeping it all consistent and to move everything at once.